beaker-hostgenerator.gemspec

```diff
@@ -19,6 +19,8 @@ eos
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.required_ruby_version = Gem::Requirement.new('>= 2.4')
```

Comment:
this is what our CI currently tests on, so I assume that's the oldest supported version.